packages/kotti-ui/source/kotti-filter/components/FilterButtonLink.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="kt-filter-button-link">
+		<span @click="handleClick">
+			<i class="yoco" v-text="icon" /> {{ label }}
```

Comment:
If this is not `required`, the icon shouldn’t be rendered when `icon` isn’t passed: `v-if="icon !== null"`